lib/SiftRequest.php

```diff
@@ -14,20 +14,22 @@ class SiftRequest {
     private $body;
     private $params;
     private $auth;
+    private $curl_opts;
 
     /**
      * SiftRequest constructor
      *
-     * @param string $url  Url for the HTTP request
-     * @param string $method  Method for the HTTP request
-     * @param int $timeout  Request timeout
-     * @param string $version  Version of the Sift Science API that is being called.
-     * @param array $opts  Array of optional parameters for this request --
+     * @param string $url Url for the HTTP request
```

Comment:
These should be single-spaced: http://docs.phpdoc.org/references/phpdoc/tags/param.html